hathor/transaction/storage/transaction_storage.py

```diff
@@ -988,6 +998,58 @@ def _topological_fast(self) -> Iterator[BaseTransaction]:
         yield from cur_blocks
         yield from cur_txs
 
+    def _topological_fast2(self) -> Iterator[BaseTransaction]:
+        import heapq
+        from dataclasses import dataclass, field
+        cur_timestamp = settings.GENESIS_TIMESTAMP
+
+        @dataclass(order=True)
+        class Item:
+            timestamp: int
+            is_transaction: bool
```

Comment:
Add a comment explaining that `False < True`.

Reply:
Done.

Reply:
Thanks!